api/lua/pinnacle/tag.lua

```diff
@@ -194,6 +194,37 @@ function tag.remove(tags)
     end
 end
 
+---Move existing tags to the specified output.
+---
+---#### Example
+---```lua
+---local tag_to_move = Tag.get("1")
+---Tag.switch_output(Output.get_by_name("HDMI-1"), tag_to_move)
+---```
+---
+---@param output pinnacle.output.OutputHandle The output to add tags to.
+---@param ... string The names of the new tags.
+---
+---@overload fun(output: pinnacle.output.OutputHandle, tag_names: string[])
+function tag.switch_output(output, ...)
+    local tag_names = { ... }
+    if type(tag_names[1]) == "table" then
+        tag_names = tag_names[1] --[=[@as string[]]=]
+    end
+
+    local response, err = client:pinnacle_tag_v1_TagService_SwitchOutput({
+        output_name = output.name,
+        tag_names = tag_names,
+    })
```

Comment:
For this part, I you can copy the way tag.remove (the previous function) handles its array of TagHandle and call the client. It should be similar (except `SwitchOutput` has `output_name` as an additional parameter)